modules/claims_api/app/sidekiq/claims_api/service_base.rb

```diff
@@ -61,11 +61,15 @@ def save_auto_claim!(auto_claim, status)
 
     def set_evss_response(auto_claim, error)
       auto_claim.status = ClaimsApi::AutoEstablishedClaim::ERRORED
-      auto_claim.save!
+      auto_claim.evss_response ||= []
 
-      auto_claim.evss_response = error&.original_body
+      if error&.original_body.present?
+        error.original_body.each { |e| auto_claim.evss_response << e }
+      elsif error&.errors.present?
+        error.errors.each { |e| auto_claim.evss_response << e }
+      end
 
-      save_auto_claim!(auto_claim, auto_claim.status)
+      auto_claim.save!
     end
 
```

Comment:
memoizes evss_response so we don't over write it

Reply:
I thought that save was in place in case there was an error somewhere in lines 66 - 70 so the status would still get saved as `errored`.  In this version the status would not get saved as errored if something went wrong in lines 66 - 70 would it?  This could also be addressed if we moved the status setting and saving back into it's own method and did that before doing this part.

Reply:
Specifically I think using `set_errored_state_on_claim(auto_claim)` on lines 61 and 70, before setting the `evss_response` in the PDF generator (and anywhere else that order occurs) would fix this possibility of not getting the state saved in the event some unknown/new issue breaking that response setting.